src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -157,6 +163,81 @@ public boolean isPinned()
         return getTextChannel().clearReactionsById(getId());
     }
 
+    @Nonnull
+    @Override
+    public RestAction<Void> removeReaction(Emote emote)
+    {
+        return channel.removeReactionById(getId(), emote);
+    }
+
+    @Nonnull
+    @Override
+    public RestAction<Void> removeReaction(Emote emote, User user)
+    {
+        return getTextChannel().removeReactionById(getIdLong(), emote, user);
+    }
+
+    @Nonnull
+    @Override
+    public RestAction<Void> removeReaction(String emote)
+    {
+        return channel.removeReactionById(getId(), emote);
+    }
+
+    @Nonnull
+    @Override
+    public RestAction<Void> removeReaction(String emote, User user)
+    {
+        return getTextChannel().removeReactionById(getId(), emote, user);
+    }
+
+    @Nonnull
+    @Override
+    public ReactionPaginationAction retrieveReactionUsers(Emote emote)
+    {
+        Checks.notNull(emote, "Emote");
+
+        MessageReaction reaction = this.reactions.stream()
+            .filter(r -> r.getReactionEmote().isEmote() && r.getReactionEmote().getEmote().equals(emote))
+            .findFirst().orElse(null);
+        return new ReactionPaginationActionImpl(reaction);
+    }
+
+    @Nonnull
+    @Override
+    public ReactionPaginationAction retrieveReactionUsers(String emote)
+    {
+        return retrieveReactionUsers(MessageReaction.ReactionEmote.fromUnicode(emote, getJDA()).getEmote());
```

Comment:
This is not how it works. Emoji != Emote

Reply:
Should be fixed now